src/homework1/task13.py

```diff
@@ -0,0 +1,20 @@
+# 13 lines: Unit testing with unittest
+import unittest
+
```

Comment:
2 lines between code and import (1 in the previous case). It should be consistent